bugwarrior/services/github.py

```diff
@@ -49,6 +49,17 @@ def get_issues(self, username, repo):
             username=username, repo=repo)
         return self._getter(url)
 
+    def get_issue_from_path(self, issue_path):
+        """Get issue from issue_path."""
+        # TODO: test new scheme of issue_path
+        regex = r'(^[^/]+)(/)([^/]+)(/issues/[0-9]+)'
+        if not re.match(regex, issue_path):
+            log.error("issue path given in config is invalid")
+            raise ValueError
```

Comment:
Here, just do ``raise ValueError("issue path %s given in config is invalid" % issue_path)`` which will give the user more information.

Reply:
Good idea. I consider this review resolved, it's just that we need to decide whether to try to check this regex match in `__init__` or not.. See my previous comment.